cmd/zapi/cmd/post/post.go

```diff
@@ -0,0 +1,125 @@
+package post
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"sync/atomic"
+	"time"
+
+	"github.com/brimsec/zq/cmd/zapi/cmd"
+	"github.com/brimsec/zq/cmd/zapi/format"
+	"github.com/brimsec/zq/pkg/display"
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/mccanne/charm"
+)
+
+var Post = &charm.Spec{
+	Name:  "post",
+	Usage: "post [options] path...",
+	Short: "post log file(s) to a space",
+	New:   NewPost,
+}
```

Comment:
Nit: For symmetry with pcap.go, s/Post/PostLog/, s/NewPost/NewLog/, and rename this file to log.go.

Reply:
ok

Reply:
My preference would be to merge this two commands. For now, I will do this.